pages/common/git-diff.md

```diff
@@ -25,3 +25,7 @@
 - Create a patch file:
 
 `git diff > {{target_file}}.patch`
+
+- Compare a single file between two branches / commits:
+
+`git diff {{branch_1}}..{{branch_2}} [--] {{/path/to/file}}`
```

Comment:
>Also, could you remove the preceding slash in /path/to/file making it path/to/file, please?

Yeah sure

>Are the -- optional? I thought they were required.

They are indeed optional.  I believe they are only needed to disambiguate certain scenarios or when the file contains dashes.

https://stackoverflow.com/questions/22750028/in-git-what-does-dash-dash-mean

```
$ git --no-pager diff master..git-diff-across-branches git-diff.md           
diff --git a/pages/common/git-diff.md b/pages/common/git-diff.md
index 69a9f8c..d82e4be 100644
--- a/pages/common/git-diff.md
+++ b/pages/common/git-diff.md
@@ -25,3 +25,7 @@
 - Create a patch file:
 
 `git diff > {{target_file}}.patch`
+
+- Compare a single file between two branches / commits:
+
+`git diff {{branch_1}}..{{branch_2}} [--] {{/path/to/file}}`
```
```
$ git --no-pager diff master..git-diff-across-branches -- git-diff.md
diff --git a/pages/common/git-diff.md b/pages/common/git-diff.md
index 69a9f8c..d82e4be 100644
--- a/pages/common/git-diff.md
+++ b/pages/common/git-diff.md
@@ -25,3 +25,7 @@
 - Create a patch file:
 
 `git diff > {{target_file}}.patch`
+
+- Compare a single file between two branches / commits:
+
+`git diff {{branch_1}}..{{branch_2}} [--] {{/path/to/file}}`
```